dmpconfig: {
  alpha_e: 5,
  alpha_z: 25,
  alpha_x: 1,
  beta_z: 6.25,
  kc: 0,
  n_kernel: 15
}